tools/fetch_source_files.py

```diff
@@ -365,7 +446,9 @@ def run(*services: t.Union[Gui, Rest, Orchestrator], **kwargs) -> t.Optional[t.U
 
 # Generate Pipfile from package dependencies from all repositories
 pipfile_path = os.path.join(ROOT_DIR, "Pipfile")
-pipfile_message = "WARNING: Package versions mismatch in Pipfiles - Pipfile not updated."
+pipfile_message = (
```

Comment:
I would keep it inlined.

Reply:
That was not me!.. for the code formatter.